### DISCUSSION TITLE
RoadRunner vs  bolt + loco deck

### DISCUSSION THREAD

**Discussion by afwilkin** [bitcraze/discussions#1125] [Q&A] [answered]
Was looking to integrate the roadrunner, but saw it was not produced anymore. Saw that the bolt + loco deck was the next best alternative.
However, I started comparing the two and had a question.

Does the loco deck know its position in space? It appeared that the roadrunner did with the IMU whereas the bolt + loco deck requires the actual CrazyFlie to do the positioning. 

Please let me know if I am off base. Thank you!

**Comment by knmcguire** [comment, marked as answer]
hi!

The [bolt](https://store.bitcraze.io/collections/kits/products/crazyflie-bolt-1-1) is a 'crazyflie' but made to support brushless motors and bigger platforms. You will be able to get positioning with just the bolt and the [locodeck](https://www.bitcraze.io/products/loco-positioning-deck/).

**Reply by afwilkin** [reply]
Maybe I should have read the description instead of just assuming it was JUST Power management board from the image: 
 "The Crazyflie Bolt 1.1 is a Crazyflie 2.X compatible flight controller"

@knmcguire apologies for your quick response, and thank you SO MUCH!

**Reply by knmcguire** [reply]
Ah yeah I can get where that confusion comes from. You probably was confusing the Bolt with the BiqQuaddeck https://www.bitcraze.io/products/bigquad-deck/ 😄
